addons/web/models/ir_http.py

```diff
@@ -176,7 +176,7 @@ def get_frontend_session_info(self):
             'is_system': user._is_system() if session_uid else False,
             'is_public': user._is_public(),
             'is_website_user': user._is_public() if session_uid else False,
-            'user_id': user.id if session_uid else False,
+            'uid': user.id if session_uid else False,
```

Comment:
in backend session_info: `"uid": session_uid,`
here: `'uid': user.id if session_uid else False,`

What's the difference? They must be the same otherwide is will be really confusing

Reply:
I didn't think about it at first, I just replaced the key, but know what you said makes sense to me , since if we have session_uid we will use that uid otherwise it's false anyway, I will fix it , thanks